ts/features/idpay/wallet/screens/AvailableInitiativesListScreen.tsx

```diff
@@ -77,29 +78,36 @@ export const IdPayInitiativeListScreen = (props: Props) => {
   }, [dispatch, idWallet, areInitiativesInError]);
 
   return (
-    <BaseScreenComponent
-      headerTitle={TypedI18n.t("idpay.wallet.initiativePairing.navigation")}
-      goBack={true}
-    >
-      <View style={IOStyles.horizontalContentPadding}>
-        <H1>{TypedI18n.t("idpay.wallet.initiativePairing.header")}</H1>
-        <VSpacer size={16} />
-        {maskedPan && (
-          <View style={IOStyles.row}>
-            <LogoPayment name={brandToLogoPaymentMap[brand]} />
-            <HSpacer size={8} />
-            <H4>•••• {maskedPan}</H4>
-          </View>
-        )}
-        <VSpacer size={16} />
-      </View>
-      <ScrollView style={IOStyles.horizontalContentPadding}>
-        <IDPayInitiativesList
-          initiatives={idpayInitiatives}
-          idWallet={idWallet}
-        />
-        <VSpacer size={24} />
-      </ScrollView>
-    </BaseScreenComponent>
+    <>
+      <BaseScreenComponent
+        headerTitle={TypedI18n.t("idpay.wallet.initiativePairing.navigation")}
+        goBack={true}
+      >
+        <ScrollView style={styles.container}>
+          <H1>{TypedI18n.t("idpay.wallet.initiativePairing.header")}</H1>
+          <VSpacer size={16} />
+          {maskedPan && (
+            <View style={IOStyles.row}>
+              <LogoPayment name={brandToLogoPaymentMap[brand]} />
+              <HSpacer size={8} />
+              <H4>•••• {maskedPan}</H4>
+            </View>
```

Comment:
Needs more padding, something like:

```suggestion
            <View style={[IOStyles.row, { paddingVertical: 8 }]}>
              <LogoPayment name={brandToLogoPaymentMap[brand]} />
              <HSpacer size={8} />
              <H4>•••• {maskedPan}</H4>
            </View>
```